cmapPy/math/fast_cov.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import cmapPy.pandasGEXpress.setup_GCToo_logger as setup_logger
+import numpy
+
+
+logger = logging.getLogger(setup_logger.LOGGER_NAME)
+
+
+def fast_cov(x, y=None, axis=0, do_print_trace=False):
```

Comment:
Can you explain why do_print_trace instead of just leaving the intermediate steps as the verbose version of logger? Seems like if someone wanted that argument to be true they'd just set their logger appropriately

Reply:
So my reasoning was that debuggers in other systems have a "trace" level that is even more "noisy" than debug.  And I figured that printing out all the intermediate math is really a trace level of logging - I can imagine wanting to be able to run code with the debug level of logging in general, but not have to see the entirety of every matrix and intermediate math step printed out.  If there were a python logging level lower than debug I would use that, since there isn't, I would argue we should either keep the do_print_trace or delete it.

Reply:
Happy to delete but had another idea - what about making it a module-level variable that is set to False.  That way if you're actually working on the code you can turn it on, otherwise it is off?